server/src/router/BoardRouter.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { BoardService } from '../service/BoardService';
+
+export const BoardRouter = () => {
+    const router = Router();
+
+    router.get('/', async (req, res) => {
+        const boardService = BoardService.getInstance();
+        const { id } = req.user;
+        const data = await boardService.getBoardsByUserId(id);
+        res.status(200).json({ ...data });
```

Comment:
```suggestion
        res.status(200).json(data);
```
data를 풀었다가 다시 객체로 묶어주는 것보단 data를 바로 전달하는게 좋아보여요.
의도하신건가욥?

Reply:
data를 풀어주지 않으면 응답 json이 아래와 같이 한 뎁스가 더 나오더라구요..
그리고 api 명세서에 적은 결과와도 달라서 어쩔수없이 풀어주었습니다.
```
data : {
    myBoards: [],
    invitedBoards: []
}
```

Reply:
``` javascript
res.json(data); 
```
이렇게 하신거 맞을까요? 아래와 같이 테스트할 땐 뎁스가 안 나옵니당😅
![image](https://user-images.githubusercontent.com/47842964/100092749-eca45a00-2e99-11eb-980a-b6b9853ebe3c.png)

Reply:
아 생각해보니 `res.json({data});` 이렇게 썼었었네요.. 그러니 뎁스가 하나 더 생길 수 밖에 없었네요..ㅠ
수정하겠습니다; ㅁ;

Reply:
commit eb06294
리뷰해주신 부분 모두 수정했습니다:)